x-pack/docs/en/rest-api/watcher/put-watch.asciidoc

```diff
@@ -68,7 +68,15 @@ A watch has the following fields:
 
 | `throttle_period` | The minimum time between actions being run, the default
                       for this is 5 seconds. This default can be changed in the
-                      config file with the setting `xpack.watcher.throttle.period.default_period`.
+                      config file with the setting
+                      `xpack.watcher.throttle.period.default_period`. If
```

Comment:
Hey i did a quick eyeball of the code here and it looks like this is not correct. The code is basically "last one wins" due to the for loop + non final variable assignment when parsing the watch.

So if you have 
```
"throttle_period_in_millis": "7000",
"throttle_period": "5s",
```
the GET watch API returns `throttle_period_in_millis":5000,`

and if you have the opposite, 
```
"throttle_period": "5s",
"throttle_period_in_millis": "7000",
```
the GET watch API returns `throttle_period_in_millis":7000,`

Reply:
Here is the code im talking about for reference, 

https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/watch/WatchParser.java#L159-L163

Reply:
Tricky! I did look at that code, but I incorrectly assumed the first matching `else if` would win.

I reversed the override statements with cfe41d9cb17ef082fe8f314779ad5f9402bafa93.

Thanks for catching this @hub-cap.

Reply:
So its not about one overriding the other, its about which one is declared first in the watch. if you declare both, the last one overrides the first one, so you can see in the example above that when i declared them both, which i did in 2 watches, the output was different based on which order they were declared in.

Reply:
🤦‍♂  Sorry about that. Guess I'm feeling dense today.

Corrected again with deaef45ab4202c50b15d6acac4abfaa036ecb119.